### PR TITLE
Update zkyoto docs

### PR DESCRIPTION
**Pull request by fiexer**
- Update Docusaurus to latest version
- Removed everything related to zKyoto & zKatana
- Removed everything related to bridging to zkEVM

### CODE REVIEW

File: docs/build/introduction/astar_family.md [hunk 1]
```diff
@@ -33,40 +26,26 @@ The Shibuya native token symbol is SBY.
...
-### zKyoto (Sepolia)
+### zKyoto (Sepolia) (Deprecated)
 
-zKyoto is a Validium-based network connected to the Ethereum Sepolia testnet and Polygon AggLayer. It is the testing and proving ground for dApps deployed on the Astar zkEVM.
+zKyoto was connected to the Ethereum Sepolia testnet and Polygon AggLayer. Testing ground for dApps deployed on the Astar zkEVM. This testnet has now been transitioned to [Soneium Minato](https://docs.soneium.org/).
 
 ## Mainnets
 
-Astar has three mainnets:
+Astar Collective with it's asset supports three mainnets:
```
Comment (Gunit2481) [inline docs/build/introduction/astar_family.md:35]: _Astar Collective with **its** asset supports three mainnets:_ 

or

_Astar Collective, with the ASTR token, supports three mainnets:_

File: docs/learn/networks.md [hunk 2]
```diff
@@ -34,31 +25,26 @@ The Shibuya native token symbol is SBY.
...
 ## Mainnets
 
-Astar has three mainnets. One parachain on Kusama, One parachain on Polkadot, and a zkEVM-based network on Ethereum Layer 2.
-
-### Shiden Parachain (Substrate)
-
-Shiden is a parachain connected to the Kusama Relay Chain, and used to deploy and test new releases of Astar runtime in a live production (canary) environment. Shiden is not considered a testnet since SDN has its own tokenomics and value, but is used to validate and stabilize new software releases and upgrades for Astar Network.
-
-The Shiden native token symbol is SDN.
+Astar Collective with it's asset supports three mainnets:
```
Comment (Gunit2481) [inline docs/learn/networks.md:34]: Same here.

**Review by Gunit2481 — CHANGES_REQUESTED** [review]
Left 2 comments, otherwise LGTM! 

Thank you. 🙏

File: docs/build/introduction/astar_family.md (hunk 1 above)
Comment (fiexer) [reply docs/build/introduction/astar_family.md:35]: LGTM

File: docs/learn/networks.md (hunk 2 above)
Comment (fiexer) [reply docs/learn/networks.md:34]: LGTM

**Review by Gunit2481 — APPROVED** [review]

**Pull request merged**